dev-server/mock-routes.js

```diff
@@ -23,7 +23,7 @@ app.get('/user/profile', (req, res) => {
     });
 });
 
-app.get('/familie-ba-sak/api/fagsak/1/vedtak-html', (req, res) => {
+app.get('/familie-ba-sak/api/behandling/1/vedtak-html', (req, res) => {
```

Comment:
Wont this break the other fetches?

Reply:
Dont quite understand... What do you mean?

Reply:
Since the test will try to fetch a fagsak, but now it is only able to fetch a behandling...

Reply:
You mean the tests.spec.js? The url "http://ci-test-server:8000/fagsak/1/vedtak" points to the frontend page, whose path did not change. But the page will now call the back api at changed path: "/familie-ba-sak/api/behandling/1/vedtak-html", which is consistent with the url here

Reply:
Yes, but the app fetches the fagsak (with id 1) once the page loads and when the request is loaded it will fail with a 404.

Reply:
But I did not change the router for "/familie-ba-sak/api/fagsak/1". I tried yarn start:mock and it did not fail. Any misunderstanding here?

Reply:
Oh, right! I see it now 👍 Sorry!